src/ProtonPlayer.js

```diff
@@ -207,24 +205,31 @@ export default class ProtonPlayer {
         this._onPlaybackProgress(progress);
       }, 250);
 
-      return clip.play() || Promise.resolve();
+      clip.play();
+      return this.audioInterface.play(clip);
```

Comment:
Having to call `.play()` on 2 different objects reads a bit weird. Would it be possible to make the `audioInterface.play` method to trigger the `play` method on the `clip` ? Or maybe rename the `clip.play` method to something else? Like `clip.markAsPlaying` ?